changelog.d/20240416_162659_arbrandes_no_workers_in_dev.md

```diff
@@ -0,0 +1,2 @@
+- 💥[Feature] Use Docker compose profiles to control services. (by @arbrandes) -->
+- [Fix] Don't start Celery workers in dev mode, as they're never used. (by @arbrandes) -->
```

Comment:
We would need some sort of guidelines to understand how the workers can be setup in dev mode, especially now that we are using docker compose profiles.